packages/flutter_driver/lib/src/driver/timeline_summary.dart

```diff
@@ -86,8 +86,6 @@ class TimelineSummary {
 
   /// Encodes this summary as JSON.
   Map<String, dynamic> get summaryJson {
-    final SceneDisplayLagSummarizer sceneDisplayLagSummarizer = _sceneDisplayLagSummarizer();
```

Comment:
Is `_sceneDisplayLagSummarizer` now dead code?

Reply:
I'm assuming that it will be revived when the `SceneDisplayLagSummarizer` implementation is updated to match the engine's new event format.

@iskakaushik

Reply:
Can we remove it for now if it's dead though?  It's in source control.

Reply:
Done